docs/content/NavList.mdx

```diff
@@ -0,0 +1,536 @@
+  <summary>Rendered HTML</summary>
+
+```html
+<nav aria-label="Security">
+  <ul role="list">
+    <li><a href="/" aria-current="page">Overview</a></li>
+    <li><a href="/policy">Security policy</a></li>
+    <li><a href="/advisories">Security advisories</a></li>
+  </ul>
+</nav>
+```
+
+</details>
+
+### With groups
+
+```jsx
+<NavList>
+  <NavList.Group title="Overview">
+    <NavList.Item href="/getting-started" current>
+      Getting started
+    </NavList.Item>
+  </NavList.Group>
+  <NavList.Group title="Components">
+    <NavList.Item href="/Avatar">Avatar</NavList.Item>
+    <NavList.Item href="/Button">Button</NavList.Item>
+    <NavList.Item href="/Label">Label</NavList.Item>
+  </NavList.Group>
+</NavList>
+```
+
+<details>
+  <summary>Rendered HTML</summary>
+
+```html
+<nav>
+  <ul role="list">
+    <li>
+      <div role="presentation" id="generated-id-1">Overview</div>
+      <ul role="list" aria-labelledby="generated-id-1">
+        <li><a href="/getting-started" aria-current="page">Getting started</a></li>
+      </ul>
+    </li>
+    <li>
+      <div role="presentation" id="generated-id-2">Components</div>
+      <ul role="list" aria-labelledby="generated-id-2">
+        <li><a href="/Avatar">Avatar</a></li>
+        <li><a href="/Button">Button</a></li>
+        <li><a href="/Label">Label</a></li>
+      </ul>
+    </li>
+  </ul>
+</nav>
+```
+
+</details>
+
+### With sub-items
+
+```jsx
+<NavList>
+  <NavList.Item href="/branches">Branches</NavList.Item>
+  <NavList.Item href="/tags">Tags</NavList.Item>
+  <NavList.Item>
+    Actions
+    <NavList.SubItem href="/actions">General</NavList.SubItem>
+    <NavList.SubItem href="/actions/runners">Runners</NavList.SubItem>
+  </NavList.Item>
+</NavList>
+```
+
+<details>
+  <summary>Rendered HTML</summary>
+
+```html
+<nav>
+  <ul role="list">
+    <li><a href="/branches">Branches</a></li>
+    <li><a href="/tags">Tags</a></li>
+    <li>
+      <button id="generated-id-1" aria-expanded="false" aria-controls="generated-id-2">
+        <span>Actions</span>
+        <span><svg aria-hidden="true">...</svg></span>
+      </button>
+      <ul role="list" id="generated-id-2" aria-labelledby="generated-id-1">
+        <li><a href="/actions">General</a></li>
+        <li><a href="/actions/runners">Runners</a></li>
+      </ul>
+    </li>
+  </ul>
+</nav>
+```
+
+</details>
+
+### With a divider
+
+```jsx
+<NavList>
+  <NavList.Item href="/">Dashboard</NavList.Item>
+  <NavList.Item href="/pulls">Pull requests</NavList.Item>
+  <NavList.Item href="/issues">Issues</NavList.Item>
+  <NavList.Divider />
+  <NavList.Item href="/marketplace">Marketplace</NavList.Item>
+  <NavList.Item href="/explore">Explore</NavList.Item>
+</NavList>
+```
+
+<details>
+  <summary>Rendered HTML</summary>
+
+```html
+<nav>
+  <ul role="list">
+    <li><a href="/">Dashboard</a></li>
+    <li><a href="/pulls">Pull requests</a></li>
+    <li><a href="/issues">Issues</a></li>
+    <div role="separator"></div>
```

Comment:
I think you need `aria-hidden="true"` here as well

Reply:
I'm curious what purpose `role` would serve if `aria-hidden` is set to `true` cc @ichelsea

Reply:
Good question, I think I added that based on the [a11y audit](https://github.com/github/primer/issues/651) but didn't research it myself.

Reply:
cc: @alliethu I'm not entirely sure the nuances on this one